src/App.js

```diff
@@ -1,12 +1,17 @@
 import React from "react";
-import SignIn from "./components/SignIn";
+import SignUp from "./components/SignUp";
 import "./App.css";
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+
+const theme = createTheme();
 
 function App() {
   return (
-    <div className="App">
-      <SignIn />
-    </div>
+    <ThemeProvider theme={theme}>
+      <div className="App">
+        <SignUp />
```

Comment:
That's not what we see first. First Login page and only in case we haven't create an account you have a link to a Sign up page. Also please make custom routes for both Login and Sign up pages. Thanks

Reply:
I was just entering the component in the `App.js` file just to see it on my browser. We do not have to render it now. This component rendering can be deleted, not important. Once the person is done with routes, they could add it to the specific route.

Reply:
We have to render them but don't worry me or Xeniya can handle with routes.